src/Raven.Server/Json/BlittableJsonTextWriterExtensions.cs

```diff
@@ -689,11 +689,11 @@ public static async Task<(long NumberOfResults, long TotalDocumentsSizeInBytes)>
                     (w, c, spatialShape) => w.WriteSpatialShapeResult(c, spatialShape));
             }
 
-            if (result.RaftCommandIndex.HasValue)
+            if (result.IndexDefinitionRaftIndex.HasValue)
```

Comment:
@arekpalinski Shouldn't we re-add here the AutoIndexCreationRaftCommandIndex (and in the other place)?

Reply:
Yes, added in https://github.com/ravendb/ravendb/pull/15844/commits/b7a695f9de930240aedc4a3e0645657924ec235d